mountpoint-s3/src/cli.rs

```diff
@@ -432,7 +460,7 @@ where
     );
 
     if args.foreground {
-        init_logging(args.logging_config()).context("failed to initialize logging")?;
+        init_logging(args.logging_config(/* is_fork_parent= */ false)).context("failed to initialize logging")?;
```

Comment:
I am not a fan of this style to document a bool. But in this case maybe we can avoid the parameter entirely (see other comment).